dspace-sword/src/main/java/org/purl/sword/client/ClientFactory.java

```diff
@@ -24,8 +24,8 @@ public ClientFactory() {
         // configure the logger from the property file. The GUI client will
         // reload these properties if it is set to capture the output and
         // display it in a panel.
-        PropertyConfigurator.configure(this.getClass().getClassLoader()
-                                           .getResource(ClientConstants.LOGGING_PROPERTY_FILE));
+        // Replace with log4j 2: PropertyConfigurator.configure(this.getClass()
+        // .getClassLoader().getResource(ClientConstants.LOGGING_PROPERTY_FILE));
```

Comment:
Commented out code that needs replaced

Reply:
I think we shouldn't be tinkering with this at all.  Just let Log4J2 find the configuration on the classpath via its normal search.  If not found, you get basic console logging.

Reply:
That seems reasonable. If this isn't really needed, let's simply remove it.